CHANGELOG.md

```diff
@@ -7,6 +7,11 @@
 - Added `template` validation in `PromptTemplate` class upon construction.
 - `custom_prompt` arg is now converted to `Text2CypherTemplate` class within the `Text2CypherRetriever.get_search_results` method.
 - `Text2CypherTemplate` and `RAGTemplate` prompt templates now require `query_text` arg and will error if it is not present. Previous `query_text` aliases may be used, but will warn of deprecation.
+- Fix bug in `Text2CypherRetriever` using `custom_prompt` arg where the `search` method would not inject the `query_text` content.
```

Comment:
3 duplicated items (they are at the end of this list as well)

Reply:
https://github.com/neo4j/neo4j-graphrag-python/pull/142/commits/e131ed61a16e8d6843fa2f528fe6b755bc23fe29